contracts/AutomatedFunctionsConsumer.sol

```diff
@@ -26,9 +26,12 @@ contract AutomatedFunctionsConsumer is FunctionsClient, ConfirmedOwner, Automati
   uint256 public s_updateInterval;
   uint256 public s_lastUpkeepTimeStamp;
   uint256 public s_upkeepCounter;
+  uint256 public s_requestCounter;
```

Comment:
What is this counter for? Just to check how many responses vs requests we had so far?

Reply:
Yeah. Not necessary, but might be useful so I added it. I like to use it for heartbeats.